src/main/java/frc/pathrecogonizer/VisionFieldLayoutRecognizer.java

```diff
@@ -0,0 +1,83 @@
+package frc.pathrecogonizer;
+
+import java.util.List;
+import java.util.ArrayList;
+
+ public class VisionFieldLayoutRecognizer
+    implements PixyFieldPoseResolverInterface{
+
+
+    //Cannot instantiate the type List if I say new List<DistanceRange>, I get this error
+    private List<DistanceRange> distanceRanges;
+    
+    public VisionFieldLayoutRecognizer() {
+      setupInitialDistanceRanges();
+    }
+
+    public void setupInitialDistanceRanges(){
+        
+      this.distanceRanges = List.of(new DistanceRange(10.0,8.50, VisionPathName.PathRedA),
+      new DistanceRange(16.30,16.10, VisionPathName.PathRedB),
+      new DistanceRange(10.40,10.30, VisionPathName.PathBlueA),
+      new DistanceRange(17.80, 17.50, VisionPathName.PathBlueB)
+      );
+
+  }
+  private double ComputeAverageDistance(List<FieldVisionInput> fieldVisionInputParameter){
+    List<Integer> pixels = new ArrayList<Integer>();
+    double focalLength = 11.33;//pixels = 3mm
```

Comment:
you didnt change these to all-caps constants!
declare these as public static final double FOCAL_LENGTH in the class header!!!!